boards/arm/nucleo_f756zg/doc/nucleof756zg.rst

```diff
@@ -0,0 +1,216 @@
+.. _nucleo_f756zg_board:
+
+ST Nucleo F756ZG
+################
+
+Overview
+********
+
+The STM32 Nucleo-144 boards offer combinations of performance and power that
+provide an affordable and flexible way for users to build prototypes and try
+out new concepts. For compatible boards, the SMPS significantly reduces power
+consumption in Run mode.
+
+The Arduino-compatible ST Zio connector expands functionality of the Nucleo
+open development platform, with a wide choice of specialized Arduino* Uno V3
+shields.
+
+The STM32 Nucleo-144 board does not require any separate probe as it integrates
+the ST-LINK/V2-1 debugger/programmer.
+
+The STM32 Nucleo-144 board comes with the STM32 comprehensive free software
+libraries and examples available with the STM32Cube MCU Package.
+
+Key Features
+
+    - STM32 microcontroller in LQFP144 package
+    - Ethernet compliant with IEEE-802.3-2002 (depending on STM32 support)
+    - USB OTG or full-speed device (depending on STM32 support)
+    - 3 user LEDs
+    - 2 user and reset push-buttons
+    - 32.768 kHz crystal oscillator
+
+    Board connectors:
```

Comment:
This should be a list item, with a nested list, and the remaining items should be list items too:

    - Board connectors:

      - USB with Micro-AB
      - SWD
      - Ethernet RJ45 (depending on STM32 support)
      - ST Zio connector including Arduino* Uno V3
      - ST morpho

    - Flexible power-supply options: ST-LINK USB VBUS or external sources.
    - On-board ST-LINK/V2-1 debugger/programmer with USB re-enumeration
      capability: mass storage, virtual COM port and debug port.
    - Comprehensive free software libraries and examples available with the
      STM32Cube MCU package.
    - Arm* Mbed Enabled* compliant (only for some Nucleo part numbers)